tools/bootstrap-cluster/bootstrap-cluster.sh

```diff
@@ -316,6 +330,27 @@ run_k0s_worker_daemon() {
   sudo $K0S_EXECUTABLE start
 }
 
+check_node_disk_size() {
+  disk_size=$(df -h | awk '/ \/$/ { print $4 }')
+
+  # Extract the numeric value and unit
+  numeric_value=$(echo "$disk_size" | sed 's/[A-Za-z]//g')
+  unit=$(echo "$disk_size" | sed 's/[0-9.]//g')
+
+  # Convert units to GiB
+  case "$unit" in
+      T) numeric_value=$(awk "BEGIN { print $numeric_value * 1024 }") ;;
+      M) numeric_value=$(awk "BEGIN { print $numeric_value / 1024 }") ;;
+      K) numeric_value=$(awk "BEGIN { print $numeric_value / 1024 / 1024 }") ;;
+  esac
+
+  if [ "$numeric_value" -lt 100 ]; then
+      bold "Warning: This node does not have enough disk space. Please consider expanding your disk size."
```

Comment:
Warning: Node does not have enough disk space on the root(/) volume. Please consider expanding your disk size.